### ISSUE TITLE
Part 1: "Creating An App" section exactly the same for Blaze/React/Angular

### ISSUE THREAD

**Issue by ppotoplyak** [meteor/tutorials#195]
Teaching my 12 year old web basics and asked him to change the "Click Me" button counter to increment by 3.  Asked him to specifically use the "React" version.  He then pointed out that all three tutorial flavors create the same exact app on the "Creating an App" step, which is not really intuitive or explicitly stated.

**Comment by ppotoplyak** [comment]
IMO most beginners would expect the app created in the first step to be of the flavor of tutorial chosen.

**Comment by sebakerckhof** [comment]
You are right. Meteor 1.8 introduces a way to directly start with a react app, but the tutorial is not updated for this yet (although it's mentioned in the second step).
The tutorials are currently being rewritten, so this should be resolved soon.
cc @leonardoventurini

**Comment by leonardoventurini** [comment]
Yep, they are being rewritten, we are taking steps to make it friendlier to beginners as well.
